website/integrations/template/service.md

```diff
@@ -20,6 +20,10 @@ The following placeholders are used in this guide:
 - `service.company` is the FQDN of the Service installation. (Remove this for SaaS)
 - `authentik.company` is the FQDN of the authentik installation.
 
+:::note
+This documentation lists only the settings that have been changed from their default values. Be aware that any changes other than those explicitly mentioned in this guide could cause issues accessing your application.
```

Comment:
```suggestion
This documentation lists only the settings that need to be changed from their default values. Be aware that any changes other than those explicitly mentioned in this guide could cause issues accessing your application.
```

Reply:
Let's avoid the passive voice, and also make it clear that we are talking about the values we instruct them to change... not some random values that the system changed, lol.